### PR TITLE
MEDTOG-425 Build for Massive

### PR DESCRIPTION
**Pull request by skogtake**
Added webpack config for building the project without certain modules: Default UI, Dash, Cast and shaka-player.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -24,6 +24,8 @@
   try {
     var kalturaPlayer = KalturaPlayer.setup(config);
     kalturaPlayer.loadMedia({entryId: '0_wifqaipd'});
+    document.querySelector('.kaltura-player-container').appendChild(kalturaPlayer.getView());
+    window.kalturaPlayer = kalturaPlayer;
```
Comment (elsassph) [inline src/index.html:28]: Not needed, the scope is `window`

File: package.json [hunk 2]
```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "clean": "rm -rf ./dist",
     "prebuild": "npm run clean",
+    "dev:ovp:massive": "webpack-dev-server --config webpack.config.massive.ovp.js",
```
Comment (elsassph) [inline package.json:8]: Rename `massive` into `no-ui` everywhere please.

**Review by elsassph — COMMENTED** [review]
Generally: remove all references to this work being a Massive/project=specific change but rather adding an option for a "no UI configuration".

File: package.json (hunk 2 above)
Comment (medin-n) [reply package.json:8]: It is not only UI that is excluded lets call it `minimal`

File: webpack.config.massive.ovp.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      {
+        from: '../samples/style.css',
+        to: '.'
+      }
+    ])
+  );
+}
+
+const entry = {
+  'kaltura-ovp-player': 'index.js'
```
Comment (medin-n) [inline webpack.config.massive.ovp.js:61]: `kaltura-player` instead since that is the main file in package.json

File: package.json (hunk 2 above)
Comment (medin-n) [reply package.json:8]: And squash into one commit please

**Pull request closed without merging**